app/2.0/start/quick-tour.md

```diff
@@ -93,6 +106,9 @@ with a styled `<div>` tag.
   <iframe frameborder="0" src="samples/picture-frame/index.html" width="100%" height="60"></iframe>
 </demo-tabs>
 
+* Try adding divs in index.html to observe how they are unaffected by the styles applied to the divs in picture-frame’s shadow DOM.
```

Comment:
We generally use code font and brackets for element names, and code font for filenames. "to observe how they are unaffected" is a little awkward, maybe rephrase as a question:

Try adding a `<div>` to `index.html`; is it affected by the styles in `<picture-frame>`'s shadow DOM?